Makefile

```diff
@@ -44,4 +44,4 @@ ensure_ns:
 	kubectl create ns ${DDS_K8S_NS} || :
 
 run_local_tests:
-	python3 -m unittest discover
+	python3 -m unittest discover -v
```

Comment:
I'd rather have two test targets. The new one could named be something like "run_local_tests_verbosely."

Reply:
You pointed out earlier that a crash in unit test is painfully hard to pint point to a test case. We could have a random rare crash with a data race bug. Strongly encouraging everyone to run verbose test would give us much more information if a random crash happens.

Is that enough justification to have only verbose test target?

Reply:
It's not clear to me that a developer who encounters such a crash will report it, especially when the output is not in a format to which he's accustomed. We should figure out a way to encourage that. Most of us will just run the test again and hope that the problem goes away.

That said, I'm happy with whatever you decide. 

One other possibility would be to make the default version verbose and add a "run_succinct_local_tests" target.

Reply:
Does `make run_local_tests SILENT=1` sound like a reasonable solution?

I decided implement that as a compromise to avoid duplicate targets.

Reply:
Seems fine. I'd add a comment with the syntax to run the test and get the usual xUnit dots.

For what it's worth, I don't mind multiple similar targets. I view Make as both a dependency analyzer so that our builds run faster and as a kind of executable documentation, reminding us how to accomplish various tasks.

Reply:
Also, I prefer to avoid negative tests where possible. Can you flip the branches and use ifeq?

Not a big deal…